src/app/app.module.ts

```diff
@@ -1,18 +1,34 @@
 import { NgModule } from '@angular/core';
 import { BrowserModule } from '@angular/platform-browser';
-
+import {HttpClientModule} from '@angular/common/http';
 import { AppRoutingModule } from './app-routing.module';
 import { AppComponent } from './app.component';
+import { ProductListComponent } from './components/MainMenu/product-list/product-list.component';
+import { ProductListService } from './components/product-list.service';
+import { NZ_I18N } from 'ng-zorro-antd/i18n';
+import { it_IT } from 'ng-zorro-antd/i18n';
+import { registerLocaleData } from '@angular/common';
+import it from '@angular/common/locales/it';
+import { FormsModule } from '@angular/forms';
+import { BrowserAnimationsModule } from '@angular/platform-browser/animations';
+import { DemoNgZorroAntdModule } from './ng-zorro-antd.module';
```

Comment:
Follow this pattern in imports:

First, all angular imports at the top, then a new line.
Secondly, all third party imports like ng-zorro-antd, then a new line.
Lastly, all local imports like product list service and component.